engine/Shopware/Controllers/Backend/Order.php

```diff
@@ -685,17 +701,21 @@ public function batchProcessAction()
             return;
         }
 
-        foreach ($orders as $key => $data) {
-            $orders[$key]['mail'] = null;
-            $orders[$key]['languageSubShop'] = null;
+        foreach ($orders as &$data) {
```

Comment:
Is it really necessary to change this loop to use references? I'd avoid this like hell tbh.